lib/src/endpoints/show_episodes.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2017, chances. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+part of spotify;
+
+class ShowEpisodes extends EndpointPaging {
+  @override
+  String get _path => 'v1/shows';
+
+  ShowEpisodes(SpotifyApiBase api) : super(api);
+
+  /// Get a Show's Episodes
+  ///
+  /// [offset]: Optional. The index of the first episode to return.
+  /// Default: 0 (the first object). Use with limit to get the next set of episodes.
+  ///
+  /// [limit]: Optional. The maximum number of episodes to return.
+  /// Default: 20. Minimum: 1. Maximum: 50.
+  ///
+  /// [market]: An ISO 3166-1 alpha-2 country code or the string 'from_token'.
+  /// If a country code is specified, only artists, albums, and tracks with
+  /// content that is playable in that market is returned.
+  Future<Iterable<Episode>> list(
```

Comment:
What do you think of using paging here, since you already derive this class from `EndpoingPaging`? There is an example here: https://github.com/rinukkusu/spotify-dart/blob/master/lib/src/endpoints/albums.dart#L28

Maybe something like this:

```dart
Pages<Episode> list(String showId, {String market = null}) {
    var query = _buildQuery({'market': market});
    var queryString = query.IsNotEmpty ? '?$query' : '';

    return _getPages(
        '$_path/$showId/episodes$queryString', 
        (json) => Episode.fromJson(json));
  }
```

Reply:
I implement but show this error:
`_TypeError (type 'SpotifyApiMock' is not a subtype of type 'SpotifyApi')`

Reply:
I made a little change in endpoint_paging to fix this

Reply:
LGTM